Update deprecated page to be useful (#27105)
mdn/content: files/en-us/web/html/element/html/manifest/index.md

The **`manifest`** attribute of the `<html>` element specifies a URL of an application cache manifest that is downloaded in the early stages of page load.

> **Note:** manifest-based caching mechanism has been deprecated. Use [service workers](/en-US/docs/Web/API/Service_Worker_API) instead.

> The `manifest` attribute only had an effect during early stages of page load, thus changing it via regular DOM interfaces had no effect. The deprecated {{domxref("Window.applicationCache")}} interface was used instead. The attribute functionality has been superceded by the [web app manifest](/en-US/docs/Web/Manifest), a JSON file deployed in from the {{CSSXref("head")}} of the HTML file with the {{CSSXref("link")}} with [`rel="manifest"`](/en-US/docs/Web/HTML/Attributes/rel) set.

Reviewer: ```suggestion
The `manifest` attribute only had an effect during early stages of page load, thus changing it via regular DOM interfaces had no effect. The deprecated {{domxref("Window.applicationCache")}} interface was used instead. The attribute functionality has been superceded by the [web app manifest](/en-US/docs/Web/Manifest), a JSON file deployed in from the {{HTMLElement("head")}} of the HTML file with the {{HTMLElement("link")}} with [`rel="manifest"`](/en-US/docs/Web/HTML/Attributes/rel) set.
```

Muscle memory? Same experience for me with `jsxref` ^^

Author: Thanks. D'oh!

> The `manifest` attribute only had an effect during early stages of page load, thus changing it via regular DOM interfaces had no effect. The deprecated {{domxref("Window.applicationCache")}} interface was used instead. The attribute functionality has been superceded by the [web app manifest](/en-US/docs/Web/Manifest), a JSON file deployed in from the {{HTMLElement("head")}} of the HTML file with the {{HTMLElement("link")}} with [`rel="manifest"`](/en-US/docs/Web/HTML/Attributes/rel) set.

Reviewer: I can't comment on the technical aspects of this, so I suggest asking for another review. I find this sentence a bit awkward to read, though:

```suggestion
The `manifest` attribute only had an effect during early stages of page load, thus changing it via regular DOM interfaces had no effect. The deprecated {{domxref("Window.applicationCache")}} interface was used instead. The attribute functionality has been superceded by the [web app manifest](/en-US/docs/Web/Manifest), an external JSON file associated with the HTML file through a {{HTMLElement("link")}} with [`rel="manifest"`](/en-US/docs/Web/HTML/Attributes/rel).
```

Also, is the attribute nonfunctional in other browsers, which is what the past tense implies?
